bin/scripts/config-helper.mjs

```diff
@@ -186,6 +186,11 @@ async function askRoleQuestions () {
             name: 'APP_ROLE_ARN',
             message: 'App Role ARN: arn of an existing IAM role to use for executing the MLSpace lambdas. This value must be set to an existing role because the default CDK deployment will not create one',
         },
+        {
+            type: 'input',
+            name: 'SYSTEM_ROLE_ARN',
+            message: 'System Role ARN: arn of an existing IAM role to use for executing the MLSpace system lambdas. This value must be set to an existing role because the default CDK deployment will not create one.',
```

Comment:
Honestly I didn't understand that description either - it's basically copied from the Application role description.

I'm guessing that the baked-in assumption is that our customers will always need to create their own roles? But even then this description is not strictly true either.

I'll be interested to see if anyone has a good justification for that line in the application role, because if they don't I say we remove it from both descriptions.

Reply:
Maybe ask the PM then? I am assuming they helped with the original documentation or know why we have it this way.

Reply:
She's out today so I'm not going to block on this; we can ask on Monday